examples/ASCollectionView/Sample/ViewController.m

```diff
@@ -70,8 +70,8 @@ - (void)viewDidLoad
     {
       NSLog(@"ViewController is not nil");
       strongSelf->_data = [[NSArray alloc] init];
-      [strongSelf->_collectionView performBatchUpdates:^{
-        [strongSelf->_collectionView insertSections:[[NSIndexSet alloc] initWithIndexesInRange:NSMakeRange(0, 100)]];
+      [strongSelf->_collectionNode performBatchUpdates:^{
+        [strongSelf->_collectionNode insertSections:[[NSIndexSet alloc] initWithIndexesInRange:NSMakeRange(0, 100)]];
       } completion:nil];
       NSLog(@"ViewController finished updating collectionView");
```

Comment:
Please update the comments here and a couple of lines below to use "collectionNode"